docs/api/app.md

```diff
@@ -1076,7 +1076,7 @@ details. Disabled by default.
 Set the about panel options. This will override the values defined in the app's
 `.plist` file. See the [Apple docs][about-panel-options] for more details.
 
-### `app.startAccessingSecurityScopedResource(bookmarkData)` _macOS (mas)_
+### `app.startAccessingSecurityScopedResource(bookmarkData)` _macOS_
```

Comment:
`(mas)` has a special meaning, as it indicates that this is only for Mac App Store (mas) apps. I do think that could be clearer though. I would just spell it out fully instead:

> _macOS_ (Mac App Store)

Reply:
I see.
Thought it's a typo since it's the only occurrence.

Personally i would prefer a more programmatic naming like `_macOS-appstore_`.

For the haxe type definitions i am annotating fields with these platform flags to have compile time checks for supported api calls (like: https://github.com/tong/hxelectron/blob/75fc9cbc8a69b578f083780f181cd210ad97dbe4/src/electron/main/App.hx#L277).

A proper naming scheme would make it easier to pass such flags to the compiler as they could be the same as defined in the docs.

Strings like `_platform (App Store)_` are not very handy for this.
…  also for possible future flags (for example: `windows-appstore` instead of `windows (Windows App Store)`).

Guess other platforms would profit from this too.

Reply:
> Personally i would prefer a more programmatic naming like _macOS-appstore_.

This aren't docs for programs, these are docs for humans 😄 Everything needs to be human readable without pre-processing.

The syntax 

```
_platform (restriction)_
```

Can we inferred and strictly qualified, in fact putting `mas` there makes it easier to enforce runtime / compiletime restrictions that you check `process[restriction]` (`process.mas`) before using the API if that's what you want to do.